.github/workflows/build-and-test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           TRACER_DATABASE_NAME: ${{ secrets.TRACER_DATABASE_NAME }}
           TRACER_DATABASE_HOST: ${{ secrets.TRACER_DATABASE_HOST }}
           DATABASE_URL: "postgresql://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/tracer_db" # localhost since VM
-        run: make test-tracer
+        run: ./scripts/migrate.sh && make test-tracer
```

Comment:
not sure about this, why do we have `TRACER_DATABASE_NAME` and `TRACER_DATABASE_HOST` that are pointing to prod, but then the `DATABASE_URL` is pointing to localhost?

Reply:
okay so we can just remove 
```
TRACER_DATABASE_NAME: ${{ secrets.TRACER_DATABASE_NAME }}
TRACER_DATABASE_HOST: ${{ secrets.TRACER_DATABASE_HOST }}
```
right?